src/components/pages/HomePage/HowPulsarWorks/HowPulsarWorks.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import s from './HowPulsarWorks.module.css'
+import ScreenTitle from '../ui/ScreenTitle/ScreenTitle';
+
+// SVGO breaks the illustration. To fix it, we import it as is.
+import illustrationDesktop from '!!raw-loader!./img/illustration-desktop.svg';
+import illustrationMobile from '!!raw-loader!./img/illustration-mobile.svg';
+
+import BookkeeperIcon from './img/bookkeeper.svg';
+import BrokersIcon from './img/brokers.svg';
+import ZookeeperIcon from './img/zookeeper.svg';
+import ProducerAndConsumerIcon from './img/producer-and-consumer.svg';
+import Slider from '@site/src/components/ui/Slider/Slider';
+
+const cards: CardProps[] = [
+  {
+    title: 'Producer & Consumer',
+    image: <ProducerAndConsumerIcon />,
+    children: (
+      <p>
+        A Pulsar client contains a consumer and a producer.
+        A producer writes messages on a topic.
+        A consumer reads messages from a topic and acknowledges specific messages or all up to a specific message.
+      </p>
+    )
+  },
+  {
+    title: 'Apache Zookeeper',
+    image: <ZookeeperIcon />,
+    children: (
+      <p>
+        Pulsar and BookKeeper use Apache ZooKeeper to save metadata coordinated between nodes,
+        such as a list of ledgers per topic, segments per ledger, and mapping of topic bundles to a broker.
+        It’s a cluster of highly available and replicated servers (usually 3).
+      </p>
+    )
+  },
+  {
+    title: 'Pulsar Brokers',
+    image: <BrokersIcon />,
+    children: (
+      <p>
+        Topics (i.e., partitions) are divided among Pulsar brokers.
```

Comment:
I remember @joohyukkim sent a [mail](https://lists.apache.org/thread/36mmvbjbm4g0b8khwmom7pjv5hr6ytxg) about distinguishing topics and partitions.

Perhaps it's not quite accurate here and we just use `Topics`?

cc @asafm

Reply:
It's on the homepage so we don't talk too much about the details.

I'd prefer to use "Topics" only or "Topics (partitioned or non-partitioned)".

Reply:
Guys, I agree with our confusion around Topic and Partitioned Topics. 
Let's remember the big goal here: A person who never heard of Pulsar before, manages to find 30 seconds free, whether it's on the subway, in the WC, in line in the supermarket, etc. We have to use those precious 30 seconds to convince them to *read more* - it doesn't have to be now, maybe later - but get them hooked.
Since Kafka is SO popular, I can safely assume most people who wind up reading about Pulsar know the basics of Kafka - therefore I have to use terminology known to them to explain in 30 seconds how Pulsar works.

The next section we have planned - The features page - is targeted at 5min - 15min reading time, hence uses more words and explains it better.

When I say "Topic (i.e. partitions)" I use their terminology that a topic is divided into partitions and those are the ones divided across brokers. This is what happens also for real in pulsar.